bindings/c/src/lib.rs

```diff
@@ -205,6 +205,182 @@ pub unsafe extern "C" fn libsql_disconnect(conn: libsql_connection_t) {
     // TODO close conn
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn libsql_prepare(
+    sql: *const std::ffi::c_char,
+    out_stmt: *mut libsql_stmt_t,
+    out_err_msg: *mut *const std::ffi::c_char,
+) -> std::ffi::c_int {
+    let sql = unsafe { std::ffi::CStr::from_ptr(sql) };
+    let sql = match sql.to_str() {
+        Ok(sql) => sql,
+        Err(e) => {
+            set_err_msg(format!("Wrong SQL: {}", e), out_err_msg);
+            return 1;
+        }
+    };
+    let stmt = Box::leak(Box::new(libsql_stmt { stmt: stmt { sql: sql.to_string(), params: vec![]} }));
```

Comment:
I think you just want to use this https://doc.rust-lang.org/stable/std/boxed/struct.Box.html#method.into_raw instead of leak.

Reply:
What's the difference? Box::leak seems to be widely used in this file.